LIFEContractDeployment.sol

```diff
@@ -4,14 +4,16 @@ pragma solidity ^0.5.00;
 // ----------------------------------------------------------------------------
 // 'LIFE' token contract
 //
-// Deployed to : 0xFEB02D9383C49A8373F88e82EbCecB553c1837bf

```

Comment:
Be sure to remove this from the final repository as this was the beta test coin we created not the Test coin we are using now